components/ai_chat/core/common/mojom/untrusted_frame.mojom

```diff
@@ -30,6 +30,12 @@ interface ParentUIFrame {
   // then deals with whether to display the modal or to continue opening
   // the link.
   UserRequestedOpenGeneratedUrl(url.mojom.Url url);
+
+  // Drag and drop events from the untrusted iframe to the parent frame.
+  // These allow the parent to handle drag UI states while the iframe
+  // detects the drag events.
+  DragStart();
+  DragEnd();
```

Comment:
I suspect (but I'm not 100% sure) that we can get away with just the `DragStart` event as we're showing an element over the `iframe` when the drag begins which should receive all future drag events.

Reply:
`DragStart` is definitely needed from the untrusted code handling. Despite there being an overlay on top, the events still go to the untrusted iframe so it needs to tell the parent frame that dragging started.  
You can see this if you put Finder on top of the browser with focus. Then start dragging an image file from Finder onto the iframe area. You won't see the overlay come up.

Reply:
Yep I'll get rid of that, just ran out of time today. Will ping tomorrow when it's ready to review.  Thanks for the review🙏

Reply:
Removed untrusted DragEnd handling which wasn't needed here:
af73b7d257d1c77f138549daf351c5b6c291c39a